app/src/main/java/nextstep/jwp/http/HttpResponse.java

```diff
@@ -0,0 +1,100 @@
+package nextstep.jwp.http;
+
+import java.io.File;
+import java.io.IOException;
+import java.io.OutputStream;
+import java.net.URL;
+import java.nio.file.Files;
+import java.util.*;
+import java.util.stream.Collectors;
+
+public class HttpResponse {
+    private final OutputStream outputStream;
+    private HttpStatus status;
+    private String body;
+    private Map<String, String> responseMap;
+
+    public HttpResponse(OutputStream outputStream) {
+        this.outputStream = outputStream;
+    }
+
+    public HttpResponse(Builder builder) {
+        this.outputStream = builder.outputStream;
+        this.status = builder.status;
+        this.body = builder.body;
+        this.responseMap = builder.responseMap;
+    }
+
+    public OutputStream getOutputStream() {
+        return outputStream;
+    }
+
+    public void forward() throws IOException {
+        outputStream.write(response().getBytes());
+        outputStream.flush();
+    }
+
+    private String response() {
+        List<String> values = new LinkedList<>(List.of("HTTP/1.1 " + status.number + " " + status.name + " "));
+        values.addAll(responseMap.entrySet().stream()
+                .map(entry -> entry.getKey() + ": " + entry.getValue() + " ")
+                .collect(Collectors.toList()));
```

Comment:
addAll() 메서드의 메서드 파라미터로 너무 장문의 코드가 들어가는 것 같아서 
저라면 개인적으로 아래와 같이 `forEach`를 활용했을 것 같아요 🤔😃
```suggestion
        this.responseMap.entrySet().stream()
                .map(entry -> entry.getKey() + ": " + entry.getValue() + " ")
                .forEach(value -> values.add(value));
```